### DISCUSSION TITLE
Plot create progress halts

### DISCUSSION THREAD

**Discussion by fenggnef** [Chia-Network/chia-blockchain#3063] [Q&A]
Sometimes command plot create progress halts on phase 1 (command lines stop rolling), and computing seems to stop as well (CPU\disk\RAM down to about 0%).

Then I have to press ENTER manually to continue rolling command lines one by one, till they recover rolling themselves.

Did anybody have same problem?

**Comment by tonywonghs** [comment]
It happened to mine too, not only on phase 1, any phase...

I

**Reply by fenggnef** [reply]
Yep, same to me. it will happen during any phase. What is your Memory setting '-b' ? >= 4000?
